pwa/src/templates/wooItemDetailTemplate/WOOItemDetailTemplate.tsx

```diff
@@ -100,6 +101,55 @@ export const WOOItemDetailTemplate: React.FC<WOOItemDetailTemplateProps> = ({ wo
     }
   };
 
+  const getName = (name: string) => {
+    
+    const formattedName = name.replace(/_/g, ' ')
+    console.log(formattedName);
+    
+    switch (_.upperFirst(formattedName)) {
+      case "Bevindingen":
+        return t("Findings");
+      case "Conclusies":
+        return t("Conclusions");
+      case "Functiebenaming":
+        return t("Job title");
+      case "Gedraging":
+        return t("Behavior");
+      case "Onderdeel taak":
+        return t("Part of task");
+      case "Oordeel":
+        return t("Judgement");
+      case "Opdrachtgever":
+        return t("Client");
+      case "Organisatieonderdeel":
+        return t("Organizational unit");
+      default:
+        return t(_.upperFirst(formattedName));
+    }
+  };

```

Comment:
It is inefficient to upperFirst the formatted name in the switch case, and at the default return.
It would bet better to split it up. Which would also make it slightly more readable.

For the rest its good code 👍

```suggestion
    const upperFirstName = _.upperFirst(formattedName)

    switch (upperFirstName) {
      case "Bevindingen":
        return t("Findings");
      case "Conclusies":
        return t("Conclusions");
      case "Functiebenaming":
        return t("Job title");
      case "Gedraging":
        return t("Behavior");
      case "Onderdeel taak":
        return t("Part of task");
      case "Oordeel":
        return t("Judgement");
      case "Opdrachtgever":
        return t("Client");
      case "Organisatieonderdeel":
        return t("Organizational unit");
      default:
        return t(upperFirstName);
    }
  };

```